README.md

```diff
@@ -5,13 +5,50 @@ Run an open-source project? Submit a Pull Request to add yourself to the list!
 
 Visit the website: [up-for-grabs.net](http://up-for-grabs.net/)
 
+## Add your project
+
+Each of the projects is a file in the [projects](https://github.com/dahlbyk/up-for-grabs.net/blob/gh-pages/_data/projects/) folder - to add yours, create a new file named after your project, ending in `.yml`. Ensure all spaces and special characters are replaced with `-`, to make everyone's life easier.
```

Comment:
I still think a small note that this can be done in the browser with a link to [this article]https://help.github.com/articles/creating-new-files/) is a good idea.